Build/CMake/Modules/AtomicDesktop.cmake

```diff
@@ -9,6 +9,14 @@ set (ATOMIC_LINK_LIBRARIES ${ATOMIC_LINK_LIBRARIES} LibCpuId SQLite)
 
 # Check whether the CEF submodule is available
 if (EXISTS ${CMAKE_SOURCE_DIR}/Submodules/CEF)
+    #Check if CEF got pulled
+    file(GLOB RESULT ${CMAKE_SOURCE_DIR}/Submodules/CEF)
+    list(LENGTH ${RESULT} RES_LEN)
+    if(RES_LEN EQUAL 0)
+     # DIR is empty, so init the submodule and checkout master
```

Comment:
Another thought, should be able to detect whether git is on path, if so attempt to clone submodules.  If not, should exit with a message